content/blog/getting-started-with-hpe-greenlake-api-for-virtualization.md

```diff
@@ -0,0 +1,412 @@
+
+```json
+{
+    "items": [
+        {
+            "dataOrchestratorInfo": {
+                "id": "d36122b2-xxxx-xxxx-xxxx-15d2521926f0",
+                "resourceUri": "/backup-recovery/v1beta1/data-orchestrators/d36122b2-xxxx-xxxx-xxxx-15d2521926f0",
+                "type": "backup-recovery/data-orchestrator"
+            },
+            "hypervisorManagerType": "VMWARE_VCENTER",
+            "id": "0c9acaa0-xxxx-xxxx-xxxx-eaa239798f6d",
+            "name": "cds-tme-vcenter.rtplab.nimblestorage.com",
+            "releaseVersion": "7.0.3",
+            "services": [
+                "backup-and-recovery”
+            ],
+            "state": "OK",
+            "status": "OK"
+        },
+        {
+            "dataOrchestratorInfo": {
+                "id": "d36122b2-xxxx-xxxx-xxxx-15d2521926f0",
+                "resourceUri": "/backup-recovery/v1beta1/data-orchestrators/d36122b2-xxxx-xxxx-xxxx-15d2521926f0",
+                "type": "backup-recovery/data-orchestrator"
+            },
+            "hypervisorManagerType": "VMWARE_VCENTER",
+            "id": "213a7f0d-xxxx-xxxx-xxxx-44e597c32121",
+            "name": "rtp-array392-dhci.rtplab.nimblestorage.com",
+            "releaseVersion": "7.0.3",
+            "services": [
+                "hci-manager",
+                "backup-and-recovery"
+            ],
+            "state": "OK",
+            "status": "OK"
+        }
+    ],
+    "count": 2,
+    "offset": 0,
+    "total": 2
+}
+```
+
+*T﻿he above code snippet display the response from hyper-managers API execution*.
+
+### Creation and management of a virtual machine in a cloud service provider (AWS)
+
+At the time of this release  (March 2024), the API resource to discover the Cloud Service Provider (CSP) account is not yet available as part of this released HPE GreenLake API for virtualization. However, you can still obtain the information about the account Id at the CSP’s registration on HPE Private Cloud Business Enterprise’s Cloud account menu using the legacy CSP accounts list API `GET {baseUrl}/api/v1/csp-accounts`. Assuming that your AWS account had been onboarded into either HPE GreenLake Backup Recovery or HPE Private Cloud for Business Enterprise, this account Id will be used as one of the values for the body of this creation of virtual machine API as shown below.
```

Comment:
Replace "as part of this released" with "in the HPE GreenLake API...".
Question... shouldn't "virtualization" be "Virtualization" in this first sentence?

I don't understand the last part of the last sentence in this paragraph... "of this creation of virtual machine API". Aren't they using the virtualization API to create a virtual machine?